chart/values.yaml

```diff
@@ -10,14 +10,22 @@ kafka:
   properties: ""
   truststore: ""
   keystore: ""
+  properties_file: ""
```

Comment:
Suggest we use camel-casing here to be consistent with the other entries in `values.yaml`. I.e. `propertiesFile`.

Reply:
You're right.